.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Use case
+
+<!-- What is the final use case? Describe it with words, code or even sandboxes (4ex: [codesandbox.io](https://codesandbox.io)) -->
+
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+- [x] _Bug fix_ (non-breaking change which fixes an issue)
+- [x] _New feature_ (non-breaking change which adds functionality)
+- [x] **_Breaking change_** (fix or feature that would cause existing functionality to not work as before)
+
+# Checklist
+
+<!-- Check all the items your PR meets. In general - only coverage item is allowed not to be checked. -->
+
+- [ ] Read the [Contributing Guide](https://github.com/streamich/react-use/blob/master/CONTRIBUTING.md)
+- [ ] Perform a code self-review
+- [ ] Comment the code, particularly in hard-to-understand areas
+- [ ] Update/add the documentation
+- [ ] Create the story
+- [ ] Write tests
```

Comment:
```suggestion
- [ ] Add tests for your changes
```

Reply:
I think `Update/add hook's story at Storybook` is better

Reply:
May be `Cover changes with tests`?
I'm trying to avoid "your" and other possessive pronouns, checklist sounds better without them IMO.